src/Run/Setup.cs

```diff
@@ -410,20 +410,16 @@ private string GetTool(Command commandToExecute, string os, string configPath, L
         private string GetProject(Command commandToExecute, List<string> parametersSelectedByUser)
         {
             string project = string.Empty;
-            bool moreThanOneProject = false;
-            foreach (string param in parametersSelectedByUser)
+            
+            if(parametersSelectedByUser != null)
             {
-                if (commandToExecute.Alias[param].Settings.TryGetValue("Project", out project))
+                if (parametersSelectedByUser.Count(p => commandToExecute.Alias[p].Settings.TryGetValue("Project", out project)) > 1)
```

Comment:
Your correct. I incorrectly thought this was checking >= 1. I retract my comment, Count should work for your purposes.